packages/cli/src/commands/setup/deploy/providers/flightcontrol.js

```diff
@@ -126,22 +126,133 @@ const updateGraphQLFunction = () => {
     Couldn't find graphql handler in api/src/functions/graphql.js.
     You'll have to add the following cors config manually:
 
-      cors: { origin: '*', credentials: true}
+      cors: { origin: process.env.REDWOOD_API_URL,, credentials: true}
```

Comment:
@flybayer We did need to add CORS config to the Auth function as well. I just added the `process.env.REDWOOD_API_URL` to both functions over here:
https://github.com/redwoodjs/deploy-target-ci/commit/2f8a0e72f82742472db6356f63052a0c5b14e3f3

I suggest you add the `cors: {...}` using env var to the DbAuthHandler as well in the task staring L148. What do you think?